Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -116,7 +116,7 @@ def getDatabasePatientUIDByPatientID(patientID):
       continue
     firstFile = filePaths[0]
     # Get PatientID from first file
-    currentPatientID = slicer.dicomDatabase.fileValue(firstFile, "0010,0020")
+    currentPatientID = slicer.dicomDatabase.fileValue(slicer.util.longPath(firstFile), "0010,0020")
```

Comment:
Seems like we might want to eventually move this `longPath` method to C++, perhaps in mrml or other fundamental place since this could pop up anywhere.  Maybe there's a way to wrap all IO libraries to use this longpath notation automatically?